File: src/SolucionesJava/Solucion_9.java
package SolucionesJava;

import java.util.Scanner;

public class Solucion_9 {
    public static void solucion_9() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("dame un numero para ver si es primo:");
        int numero = scanner.nextInt();

        if (numero < 2) {
            System.out.println("el numero " + numero + " no es primo");
            return;
        }

        boolean Primo = true;
        for (int i = 2; i < numero; i++) {
            if (numero % i == 0) {
                Primo = false;
                break;
            }
        }

        if (Primo) {
            System.out.println("el numero es primo");
        } else {
            System.out.println("el numero no es primo");
        }
    }
}

File: src/SolucionesKotlin/Solucion_9.kt
package SolucionesKotlin

fun Solucion_9 (){
    println("dame un numero para ver si es primo: ")
    val numero = readln().toInt()
    if(numero <2){
        println("el numero $numero no es primo")
    }
    var Primo = true
    for (i in 2..numero){
        if (numero % i == 0){
            Primo = false
            break
        }
    }
    if (Primo == true){
        println("el numero no es primo")
    }else {
        println("el numero es primo")
    }
}